### ISSUE TITLE
Details component group missing a name

### ISSUE THREAD

**Issue by gazjoy** [alphagov/govuk-frontend#1103]
I am using [SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc) on my service and have noticed that it reports a missed WCAG criteria.

`A group created with the WAI-ARIA role="group" or role="radiogroup" is missing a name.`

![image 4](https://user-images.githubusercontent.com/628064/50338339-ef5adf80-050b-11e9-89b6-72c1dd376c63.png)

Adding `aria-labelledby="<ID of label>"` to the component suppresses the issue but is the the right thing to do just to appease a tool?

When I add the `aria-labelledby` attribute VoiceOver announces the component twice. Once for the button and once for the group. See screenshot.

![image](https://user-images.githubusercontent.com/628064/50338226-93905680-050b-11e9-82b9-39aedb629981.png)

**Comment by gazjoy** [comment]
I have reported this to SiteImprove and will keep the thread updated.

**Comment by gazjoy** [comment]
@36degrees 

I had a reply from Nicolaj Dannemann at [SiteImprove](https://siteimprove.com/en-gb/) on this.

> Hi Gareth,
> 
> This is Advisory information, not an Issue per se - we are aware that the presentation of this type of issues could be clearer and it will be presented differently in the next iteration of Siteimprove Accessibility checks.
> 
> Please look at https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/understanding-techniques.html
The Important quote here is:  'Techniques are informative—that means they are not required'
> 
> We do want you to take informed action on them however, which is why we list them.

Hope that helps. 👍

**Comment by 36degrees** [comment]
@gazjoy thanks for updating us 👍 

Are you happy for this issue to be closed? It doesn't look like there's anything to fix here.

**Comment by gazjoy** [comment]
@36degrees yes, please close.
